### PR TITLE
Add important notice to 4.4 GA

### PR DESCRIPTION
**Pull request by rolfedh**

### CODE REVIEW

**Review by peterclauterbach — APPROVED** [review]
LGTM

File: modules/installing-rhv-export-some-environment-variables.adoc [hunk 1]
```diff
@@ -5,6 +5,8 @@
 [id="installing-rhv-exporting-some-environment-variables_{context}"]
 = Exporting some environment variables
 
+IMPORTANT: Using these environment variables to customize the install is a temporary workaround for a known issue (link:https://bugzilla.redhat.com/show_bug.cgi?id=1818577[*BZ#1818577*]).  This content will be removed in a future release.
```
Comment (bobfuru) [inline modules/installing-rhv-export-some-environment-variables.adoc:8]: s/will/might/
Best to not commit to future planning

s/.  This/. This/

**Review by bobfuru — COMMENTED** [review]
Couple of nits, @rolfedh - thanks.

**Review by mkumku — APPROVED** [review]
Looks good to me as well, otherwise was not clear what the section for and why it is neeeded.

**Comment by vikram-redhat** [comment]
Merging this.

**Pull request merged**